geo/src/algorithm/chamberlain_duquette_area.rs

```diff
@@ -32,31 +33,41 @@ use num_traits::Float;
 ///     (x: 0.00388383, y: 51.501574),
 /// ];
 ///
-/// let area: f64 = p.chamberlain_duquette_area();
+/// // 78,478 meters²
+/// assert_eq!(78_478., polygon.unsigned_area().round());
+/// assert_eq!(78_478., polygon.signed_area().round());
 ///
-/// assert_eq!(
-///     78478., // 78,478 meters²
-///     area.round(),
-/// );
+/// polygon.exterior_mut(|line_string| {
+///     line_string.0.reverse();
+/// });
+///
+/// assert_eq!(78_478., polygon.unsigned_area().round());
+/// assert_eq!(-78_478., polygon.signed_area().round());
 /// ```
 pub trait ChamberlainDuquetteArea<T>
 where
     T: Float + CoordinateType,
 {
-    fn chamberlain_duquette_area(&self) -> T;
+    fn signed_area(&self) -> T;
+
+    fn unsigned_area(&self) -> T;
 }
 
 impl<T> ChamberlainDuquetteArea<T> for Polygon<T>
 where
     T: Float + CoordinateType,
 {
-    fn chamberlain_duquette_area(&self) -> T {
+    fn signed_area(&self) -> T {
```

Comment:
I guess `chamberlain_duquette_signed_area`/`chamberlain_duquette_unsigned_area` would be too verbose?

Just weighing that against the ambiguity of using the same method names here - seems like a reasonable tradeoff.

Reply:
Just to note that the ([Chamberlain-Duquette algorithm](https://trs.jpl.nasa.gov/bitstream/handle/2014/41271/07-0286.pdf?sequence=1&isAllowed=y), p4) explicitly calculates the signed area – I think an unsigned variant would be confusing / no longer strictly C-D, so this is OK. We point out that it's signed in the docs, too.

Reply:
@urschrei to clarify, how do you feel about having signed/unsigned variants for C-D? if i didn't read the docs, and i saw an algorithm that returned meters, i would expect it to be unsigned, which is one reason i like having the distinction here in the method name. but i definitely don't feel strongly if this is a weird convention!

Reply:
> if i didn't read the docs, and i saw an algorithm that returned meters, i would expect it to be unsigned

That’s an interesting point. I’d prefer to be practical at the expense of strict adherence to the paper, and we do note that the original is signed.

Reply:
> I guess `chamberlain_duquette_signed_area`/`chamberlain_duquette_unsigned_area` would be too verbose?
> 
> Just weighing that against the ambiguity of using the same method names here - seems like a reasonable tradeoff as you have it.

yeah i meant to mention this when i opened the pr. i was undecided what to go with. for now, i'm going with the verbose since that's consistent with what we do for other traits (e.g. `EuclideanDistance#euclidean_distance`). https://github.com/georust/geo/pull/463/commits/8393b73e4950f03cd457f4ba8b2925d8397d2eed

we can always change this in the future if it's annoying though